zingolib/src/lightclient.rs

```diff
@@ -179,28 +179,6 @@ impl LightClient {
         self.tor_client = None;
     }
 
-    /// Returns server information.
-    // TODO: return concrete struct with from json impl
-    pub async fn do_info(&self) -> String {
-        match crate::grpc_connector::get_info(self.server_uri()).await {
-            Ok(i) => {
-                let o = json::object! {
-                    "version" => i.version,
-                    "git_commit" => i.git_commit,
-                    "server_uri" => self.server_uri().to_string(),
-                    "vendor" => i.vendor,
-                    "taddr_support" => i.taddr_support,
-                    "chain_name" => i.chain_name,
-                    "sapling_activation_height" => i.sapling_activation_height,
-                    "consensus_branch_id" => i.consensus_branch_id,
-                    "latest_block_height" => i.block_height
-                };
-                o.pretty(2)
-            }
-            Err(e) => e,
-        }
-    }
```

Comment:
instead of deleting this can we rename it to `server_info` and just pass through the result from grpc_connector::get_info? I don't think we want the grpc_connector as part of the public API it would be better if consumers can call a lightcleint method to get this information IMO